modules/ROOT/pages/activity-tracking.adoc

```diff
@@ -3,23 +3,39 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-B2B transactions often communicate contractual obligations between companies. It is therefore desirable (and often necessary) to track transmissions sent or received and store the transmission content as well as metadata. Metadata could be the date and time the transmission was sent or received, the identity of the partner, and the success or failure of processing. Anypoint Partner Manager provides a simple, secure, and integrated mechanism for activity tracking within MuleSoft flows. Some uses for this data include:
+B2B transactions often communicate contractual obligations between companies.
+It is desirable (and often necessary) to track transmissions that are sent or received and store the transmission content as well as any metadata.
+Metadata can be the date and time the transmission was sent or received, the identity of the partner, and the success or failure of processing.
+Some uses for this data include:
 
 * Non-repudiation.
 * Correlation of acknowledgments.
 * Error correction.
 * Auditing.
 
-Anypoint Partner Manager enables you to monitor B2B activity tracking. Activity is the full set of end-to-end events (including transmissions, documents, and errors) involved in processing a transmission.
+Anypoint Partner Manager allows you to monitor B2B transmission activity.
+Activity is the full set of end-to-end events involved in processing a transmission. This includes transmission metadata, messages, and errors.
 
 You can:
 
-* Filter to find the specific transmission data (for Beta?).
-* Review and monitor errors and acknowledgements.
-* Search for document metadata (for Beta?).
+* Search for document metadata (such as partner name, message type, filename, control number, endpoint, or date received).
+* View and download the payload information from the transmission.
+* Review errors and acknowledgement payloads.
+// * Filter to find the specific transmission flow data.
 
-== To Track Activity:
+== Track Activity for Message Flows:
```

Comment:
Replace with plain text, because this is the real topic, not a subtopic:

     To track B2B transmission activity: